rplugin/python3/denite/source/output.py

```diff
@@ -0,0 +1,33 @@
+# ============================================================================
+# FILE: output.py
+# License: MIT license
+# ============================================================================
+
+from .base import Base
+
+
+class Source(Base):
+
+    def __init__(self, vim):
+        super().__init__(vim)
+        self.name = 'output'
+        # why doesn't this seem to be working?
+        self.default_action = 'yank'
```

Comment:
As the comment above indicates, I could use some help determining how to correctly set the default action for this source.